openmc/tallies.py

```diff
@@ -587,8 +672,20 @@ def get_tally_xml(self):
 
         return element
 
-
     def find_filter(self, filter_type):
+        """Return a filter in the tally that matches a specified type
```

Comment:
Since this is a "get" method rather than a "find" method.
On Jun 24, 2015 5:31 PM, "Will Boyd" <wbinventor@gmail.com> wrote:

> Nope, I think the ValueError is best.
> On Jun 24, 2015 4:57 PM, "Paul Romano" <notifications@github.com> wrote:
>
>> In openmc/tallies.py
>> <https://github.com/mit-crpg/openmc/pull/400#discussion_r33211234>:
>>
>> >      def find_filter(self, filter_type):
>> > +        """Return a filter in the tally that matches a specified type
>>
>> It doesn't look like None is ever returned; rather a ValueError is
>> raised if no matching Filter is found. I'll add that in a "Raises"
>> section (unless you feel that None should be returned rather than
>> raising a ValueError).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mit-crpg/openmc/pull/400/files#r33211234>.
>>
>